api/cpp/esp-idf/slint/src/slint-esp.cpp

```diff
@@ -324,20 +320,39 @@ void slint_esp_init(slint::PhysicalSize size, esp_lcd_panel_handle_t panel,
 #endif
 )
 {
-    slint::platform::set_platform(std::make_unique<EspPlatform>(size, panel, touch, buffer1, buffer2
+    SlintPlatformConfiguration config { .size = size,
+                                        .panel = panel,
+                                        .touch = touch,
+                                        .buffer1 = buffer1,
+                                        .buffer2 = buffer2
```

Comment:
When double buffering is enabled, we should set the byte swapping to false to keep compatibility.  And byte swapping should also be implemented there.